src/plugins/intel_cpu/src/mkldnn_node.cpp

```diff
@@ -573,8 +573,15 @@ void MKLDNNNode::initSupportedPrimitiveDescriptors() {
     if (!supportedPrimitiveDescriptors.empty())
         return;
 
+    auto attr = initPrimitiveAttr();
+
     for (auto& desc : descs) {
-        auto itpd = desc.createPrimitiveDescriptorIterator(engine);
+        primitive_desc_iterator itpd;
+        if (attr) {
+            itpd = desc.createPrimitiveDescriptorIterator(engine, *(attr.get()));
+        } else {
+            itpd = desc.createPrimitiveDescriptorIterator(engine);
```

Comment:
Also I would propose to replace
const mkldnn::primitive_attr &attr with const shared_ptr<mkldnn::primitive_attr>& attr
as a second argument of createPrimitiveDescriptorIterator
since in current implementation initPrimitiveAttr gives us shared_ptr,
then we take an object from it and work with it through the reference
and then again we come back to ptr when passing it to oneDNN

Reply:
N/A anymore